src/scheduled_tasks.rs

```diff
@@ -439,6 +440,14 @@ async fn update_banned_when_expired(pool: &mut DbPool<'_>) {
       .await
       .inspect_err(|e| error!("Failed to remove community_ban expired rows: {e}"))
       .ok();
+
+      diesel::delete(
+        federation_blocklist::table.filter(federation_blocklist::expires.lt(now().nullable())),
+      )
+      .execute(&mut conn)
+      .await
+      .inspect_err(|e| error!("Failed to remove federation_blocklist expired rows: {e}"))
+      .ok();
```

Comment:
Probably a good idea to move this to its own function, since the other two are ban expires, but this is an instance block expires.